VK/VK/Services/PhotoCacheService/PhotoCacheService.swift

```diff
@@ -0,0 +1,104 @@
+// PhotoCacheService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import UIKit
+
+/// Fetch Images and Caching Images
+final class PhotoCacheService {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let imagePathName = "images"
+        static let slashName = "/"
+        static let defaultName: String.SubSequence = "default"
+    }
+
+    // MARK: - Private Properties
+
+    private let networkService = NetworkService()
+    private let cacheLifeTime: TimeInterval = 24 * 60 * 60
+    private var images = [String: UIImage]()
+
+    // MARK: - Public Methods
+
+    func photo(byUrl url: String, completion: @escaping (UIImage?) -> Void) {
+        var image: UIImage?
+        if let photo = images[url] {
+            image = photo
+            completion(image)
+        } else if let photo = getImageFromCache(url: url) {
+            image = photo
+            completion(image)
+        } else {
+            loadPhoto(byUrl: url, completion: completion)
+        }
+    }
+
+    // MARK: - Private Methods
+
+    private static let pathName: String = {
```

Comment:
```suggestion
    private static let pathName: String = {
```
марка метод а это разве метод?